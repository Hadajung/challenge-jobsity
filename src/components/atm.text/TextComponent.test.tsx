import React from 'react';
import renderer from 'react-test-renderer';

import {Text} from '../index';

it('renders title correctly', () => {
  const title = renderer.create(<Text preset="title">Title</Text>).toJSON();

  expect(title).toMatchSnapshot();
});
