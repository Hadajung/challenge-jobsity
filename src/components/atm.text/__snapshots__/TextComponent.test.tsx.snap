// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#fff",
      "fontFamily": "Mulish-Bold",
      "fontSize": 17,
      "lineHeight": 24,
    }
  }
>
  Title
</Text>
`;
